pkg/text/text.go

```diff
@@ -33,7 +33,7 @@ func Indent(s, indent string) string {
 
 // DisplayWidth calculates what the rendered width of string s will be.
 func DisplayWidth(s string) int {
-	return ansi.PrintableRuneWidth(s)
+	return uniseg.StringWidth(s)
```

Comment:
**Change Requested**

Can you add the emoji from https://github.com/cli/cli/issues/9018 into the `TestDisplayWidth` test cases?